### ISSUE TITLE
Draft an accessibility statement for Find MoJ data

### ISSUE THREAD

**Issue by markjefferson-gov** [ministryofjustice/find-moj-data#825]
Dependency: https://github.com/orgs/ministryofjustice/projects/57/views/20?sliceBy%5Bvalue%5D=Content+%26+interactions&pane=issue&itemId=79570597

When the basic accessibility check is complete, write an accessibility statement following the [GOV.UK model accessibility statement template](https://www.gov.uk/guidance/model-accessibility-statement).

Draft statement in the comment below: https://github.com/ministryofjustice/find-moj-data/issues/825#issuecomment-2740422755

**Comment by markjefferson-gov** [comment]
Draft statement is ready pending: 

* response from the accessibility specialist about our compliance with WCAG 2.1, 2.4.5 Multiple Ways
* a decision on whether the outstanding issues will be fixed before this can go live

**Comment by markjefferson-gov** [comment]
@teeceeas @YvanMOJdigital This is almost ready. There are a couple of things to note: 

* I've determined the service to be fully compliant. This assumes all the accessibility tickets are fixed first. You can go live with this without all the fixes but it will need tweaking. I'm happy to do that if you need me to.
* I've left space for a team email address, which I don't think we have at the moment. You could potentially link to the contact page instead but I'm not 100% sure of the accessibility of Slack and Teams.
* A basic accessibility check is really just a first pass to give us a baseline. I recommend you schedule in an expert audit at some point.

**Comment by YvanMOJdigital** [comment]
Bring back into sprint post formal auditing and for moving into public beta

**Comment by DeanWillicombe82** [comment]
As per above from Yvan, to be finalised post Formal Audit.

**Comment by jemnery** [comment]
# Accessibility Statement

The Ministry of Justice is committed to making its websites accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.

This accessibility statement applies to the Find MoJ Data service.

## Compliance status

[Choose one of the options below, for example, (a), (b), or (c) and delete those not applicable.]

[Select (a) only if all requirements of the technical specification are fully met without exceptions for WCAG version 2.2.]

(a) This website is fully compliant with the Web Content Accessibility Guidelines (WCAG) version 2.2 AA standard.

[Select (b) if most requirements of the technical specification are met, but with some exceptions. This means not yet fully compliant and that the necessary measures are to be taken in order to reach full compliance.]

(b) This website is partially compliant with the Web Content Accessibility Guidelines version 2.2 AA standard, due to [the non-compliance(s)] [and/or] [the exemptions] listed below.

## Non-accessible content

[This section or individual headings within this section can be deleted if not applicable.]

The content listed below is non-accessible for the following reason(s):

(a) non-compliance with the accessibility regulations

[List the non-compliance(s) of the website(s)/mobile application(s), and/or, describe which section(s)/content/function(s) are not yet compliant].

[Describe in non-technical terms, as far as possible, how the content is not accessible, including reference(s) to the applicable requirements in the relevant standards and/or technical specifications that are not met.]

(b) disproportionate burden

[List non-accessible section(s)/content/function(s) for which the disproportionate burden exemption, within the meaning of the accessibility regulations is being temporarily invoked]

(c) the content is not within the scope of the accessibility regulations

[List non-accessible section(s)/content/function(s) which is/are out of scope of the accessibility regulations].

## Preparation of this accessibility statement

This statement was prepared on [date].

[Insert date of the first preparation, or a subsequent update, of the accessibility statement following an evaluation of the websites/mobile applications to which it applies. It is recommended that an evaluation is carried out and the statement updated following a substantial revision of the website/mobile application.]

[Indicate the method used to prepare the statement. The method should be an actual evaluation of the website’s or mobile application’s compliance, such as a self-assessment done by the public sector body or an assessment carried out by a third party, for example a certification].

The statement was last reviewed on [insert date of latest review].

[It is recommended that the claims made in the accessibility statement are reviewed as regards their accuracy on a regular basis, and at least once per year. If such a review has taken place without a full evaluation of the website/mobile app, whether or not such a review has led to any changes in the accessibility statement, please indicate the date of the last such review.]

## Feedback and contact information

For feedback, questions or to report an accessibility problem with this website, you can contact us using one of the following channels:

-  Via Slack in the [#ask-data-catalogue](https://moj.enterprise.slack.com/archives/C06NPM2200N)
- Via MS Teams in the [Ask Find MoJ Data channel](https://teams.microsoft.com/l/channel/19%3ADLPKc8R2DlEnBUFYvEkV5qOcp1_FtzLNvDQjSpHDbkg1%40thread.tacv2/Ask%20Find%20MoJ%20Data?groupId=f6c3cb3b-591c-4e47-9997-25b6dc9bf5b6&tenantId=c6874728-71e6-41fe-a9e1-2e8c36776ad8&ngc=true&allowXTenantAccess=true)
- Via email at [findmojdata@justice.gov.uk](findmojdata@justice.gov.uk)

## Enforcement procedure

The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’).

If you’re not happy with how we respond to your complaint, contact the [Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/app/ask).

**Comment by murdo-moj** [comment]
https://github.com/ministryofjustice/find-moj-data/pull/1493

**Comment by DeanWillicombe82** [comment]
@jemnery @murdo-moj - This will need revising that we now have the Audit results to include no compliant issues found.

**Comment by teeceeas** [comment]
Sabrina is making some changes to the statement based on the audit
